app/models/github_status.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+class GithubStatus
+  Status = Struct.new(:context, :latest_status) do
+    def state
+      latest_status.state
```

Comment:
latest_status->status ?

Reply:
hmm I guess that could get confusing with `state` 🤷‍♂️

Reply:
Yeah.